public/javascripts/Progress/src/Progress.js

```diff
@@ -57,4 +57,39 @@ function Progress (_, $, difficultRegionIds, userRole) {
         InitializeSubmittedLabels(map, streetParams, 'null', InitializeMapLayerContainer(), data2[0])
         setRegionFocus(map, layers);
     });
+
+    function logWebpageActivity(activity){
+        var url = "/userapi/logWebpageActivity";
+        var async = false;
+        $.ajax({
+            async: async,
+            contentType: 'application/json; charset=utf-8',
+            url: url,
+            type: 'post',
+            data: JSON.stringify(activity),
+            dataType: 'json',
+            success: function(result){},
+            error: function (result) {
+                console.error(result);
+            }
+        });
+    }
+
+    function putUserOrg(e) {
+        var orgId = $(this).attr('id');
+        $.ajax({
+            async: true,
+            url: '/userapi/setUserOrg/' + orgId,
+            type: 'put',
+            success: function (result) {
+                window.location.reload();
+                logWebpageActivity("Click_module=JoiningOrg=" + orgId);
```

Comment:
Could you add a special case here so that it logs something more informative when someone is leaving an org? Ideally we could include the ID of the org that they are leaving. But at the very least we should specify in the logging that they are leaving an org instead of just saying that they are joining org 0.